common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1520,7 +1520,7 @@ def reset_problem(self, _data):
             return {
                 'success': False,
                 # Translators: 'closed' means the problem's due date has passed. You may no longer attempt to solve the problem.
-                'error': _("Problem is closed."),
+                'msg': _("Problem is closed."),
```

Comment:
Not sure in what situations a learner would see this message? I changed the due date to a date in the past, but none of my actions produced it. Also, are there any other reasons that a problem would be "closed" other than due date?